packages/teams-js/src/public/app.ts

```diff
@@ -447,32 +447,32 @@ export namespace app {
    */
   export interface Context {
     /**
-     * Info of the app
+     * Properties about the current session for your app
```

Comment:
@erikadoyle is it okay to use pronouns like this in the documentation? I know it seems like it violates typical style but when I was writing this, it felt like using "your app" was the clearest way to differentiate between the app the developer reading this documentation is implement and the containing app (Teams, Outlook, etc.)

Happy to hear alternative suggestions that are also able to clearly differentiate between the developer's app and the hosting app.

Reply:
Yes! That's great actually; [MS Docs style](https://docs.microsoft.com/en-us/style-guide/top-10-tips-style-voice) recommends using an active voice (and "you"/"your"/"you'll" etc.) for current documentation because its more casual, friendly, and directed to customers.